pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -48,3 +77,41 @@ func (pc *PackageControllerClient) ValidateControllerDoesNotExist(ctx context.Co
 	}
 	return nil
 }
+
+func (pc *PackageControllerClient) ApplySecret(ctx context.Context) error {
+	eksaAccessKeyId, eksaSecretAccessKey := os.Getenv(config.EksaAccessKeyIdEnv), os.Getenv(config.EksaSecretAcessKeyEnv)
```

Comment:
Ideally, I think we should inject these two in `NewPackageControllerClient`
It makes the business logic agnostic from how the config is read